src/test/regress/pg_regress_multi.pl

```diff
@@ -417,6 +419,7 @@ sub revert_replace_postgres
 }
 print $fh catfile($bindir, "psql")." ";
 print $fh "--variable=master_port=$masterPort ";
+print $fh "--variable=worker_2_proxy_port=$workerBehindProxyPort ";
```

Comment:
We could also change `:worker_2_port` and have it always point at the proxy, that sounds like a cleaner option.